src/Orleans.sln

```diff
@@ -271,4 +271,7 @@ Global
 	GlobalSection(NestedProjects) = preSolution
 		{0BB483EF-6401-424C-9C4F-5750B300C7FD} = {DD5D35C0-A5FE-41CC-A10D-CAAA37996479}
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		EnterpriseLibraryConfigurationToolBinariesPathV6 = packages\EnterpriseLibrary.TransientFaultHandling.6.0.1304.0\lib\portable-net45+win+wp8
```

Comment:
>This is NuGet Tool proving to us once again that it can be "trusted" right?

I don't think anyone is arguing that nuget tools should be blindly trusted.  Proper setup and dependency management helps, but nuget still gets stupid, so reviewing changes and fixing things when it does is, unfortunately, a cost of using the tool.  What is less clear is what should be considered 'wrong' with what nuget does.

Having hint path start with $(SolutionDir)packages, when the project nuget settings should be set to have this start with ..\packages, is wrong.  This could be nugget getting 'stupid' again or it could be due to something wrong in our nuget settings.

Alternatively, nuget adding \<Reference Include="Newtonsoft.Json, Version=.., Culture=.., ...\> instead of just \<Reference Include="Newtonsoft.Json"\> is less clear.

I don't suggest we try to build a case for good/bad nuget behaviors in this thread, but it would be good to build (and document) a consensus on this prior to the next major nuget update.  For this PR, I'd (unfortunately) just ask @attilah to put on his 'mindless repetitive task' cap and tediously hand fix-up the project files.  However, for his patience, we owe him a clear description of why this effort was necessary.